### ISSUE TITLE
TRIPS tests failing on Travis

### ISSUE THREAD

**Issue by johnbachman** [sorgerlab/indra#58]
```
======================================================================

FAIL: test_trips.test_phosphorylation

----------------------------------------------------------------------

Traceback (most recent call last):

  File "/home/travis/miniconda/lib/python2.7/site-packages/nose/case.py", line 197, in runTest

    self.test(*self.arg)

  File "/home/travis/build/johnbachman/indra/indra/tests/test_trips.py", line 12, in test_phosphorylation

    assert(len(tp.statements) == 1)

AssertionError: 

-------------------- >> begin captured stdout << ---------------------

All events by type

------------------

------------------

--------------------- >> end captured stdout << ----------------------

======================================================================

FAIL: test_trips.test_phosphorylation_noresidue

----------------------------------------------------------------------

Traceback (most recent call last):

  File "/home/travis/miniconda/lib/python2.7/site-packages/nose/case.py", line 197, in runTest

    self.test(*self.arg)

  File "/home/travis/build/johnbachman/indra/indra/tests/test_trips.py", line 20, in test_phosphorylation_noresidue

    assert(len(tp.statements) == 1)

AssertionError: 

-------------------- >> begin captured stdout << ---------------------

All events by type

------------------

------------------

--------------------- >> end captured stdout << ----------------------

======================================================================

FAIL: test_trips.test_phosphorylation_nosite

----------------------------------------------------------------------

Traceback (most recent call last):

  File "/home/travis/miniconda/lib/python2.7/site-packages/nose/case.py", line 197, in runTest

    self.test(*self.arg)

  File "/home/travis/build/johnbachman/indra/indra/tests/test_trips.py", line 28, in test_phosphorylation_nosite

    assert(len(tp.statements) == 1)

AssertionError: 

-------------------- >> begin captured stdout << ---------------------

All events by type

------------------

------------------

--------------------- >> end captured stdout << ----------------------

======================================================================

FAIL: test_trips.test_actmod

----------------------------------------------------------------------

Traceback (most recent call last):

  File "/home/travis/miniconda/lib/python2.7/site-packages/nose/case.py", line 197, in runTest

    self.test(*self.arg)

  File "/home/travis/build/johnbachman/indra/indra/tests/test_trips.py", line 36, in test_actmod

    assert(len(tp.statements) == 1)

AssertionError: 

-------------------- >> begin captured stdout << ---------------------

All events by type

------------------

------------------

--------------------- >> end captured stdout << ----------------------

======================================================================

FAIL: test_trips.test_actmods

----------------------------------------------------------------------

Traceback (most recent call last):

  File "/home/travis/miniconda/lib/python2.7/site-packages/nose/case.py", line 197, in runTest

    self.test(*self.arg)

  File "/home/travis/build/johnbachman/indra/indra/tests/test_trips.py", line 57, in test_actmods

    assert(len(tp.statements) == 1)

AssertionError: 

-------------------- >> begin captured stdout << ---------------------

All events by type

------------------

------------------

--------------------- >> end captured stdout << ----------------------

----------------------------------------------------------------------

Ran 252 tests in 43.860s

FAILED (failures=5)
```

**Comment by bgyori** [comment]
They have a problem with their webservice, the problem is not on our end.

**Comment by johnbachman** [comment]
OK. I fixed the preassembler doctest issue in f38865a88aec4d6, so once they fix their web service the Travis builds should start passing again.
